src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/EasyCreditResponseSaver.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Init;
+
+use Generated\Shared\Transfer\ComputopEasyCreditInitResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class EasyCreditResponseSaver extends AbstractResponseSaver
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return void
+     */
+    public function handle(QuoteTransfer $quoteTransfer)
+    {
+        $responseTransfer = $quoteTransfer->getPayment()->getComputopEasyCredit()->getEasyCreditInitResponse();
+
+        $this->handleDatabaseTransaction(function () use ($responseTransfer) {
+            $this->saveComputopDetails($responseTransfer);
+            $this->triggerEvent($this->getPaymentEntity($responseTransfer->getHeader()->getTransId()));
```

Comment:
it's not allowed to trigger OMS events inside transaction.
It will lead to unexpected results - OMS lock doesn't work when triggering event in transaction.